### PR TITLE
Initial WinUI2 GetStart project

### PR DESCRIPTION
**Pull request by mikehoffms**
This repo hasn't had a project for GetStart WinUI 2 (UWP).  This PR adds such a project.

This PR adds the resulting project that results from updating the steps in article "Get started with WebView2 in WinUI 2 (UWP) apps" https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2 - that article update is PR https://github.com/MicrosoftDocs/edge-developer/pull/2456 in "edge-developer" Docs repo.

**.sln filename:**
* MyUWP**GetStart**App.sln
   *  "GetStart" is better than "Sample", to differentiate.
   * Specifies the platform (UWP).
   * Future PRs should omit "My" and "App"; a future PR could name it "UWPGettingStarted.sln", "WinUI2GettingStarted.sln", or "WinUI2UWPGettingStarted.sln".

Existing GetStart .sln filenames:
* WPF**Sample**.sln
* WebView2**GettingStarted**.sln
* WinForms_**GettingStarted**.sln
* WinUI_**Sample**.sln

**Folder name:** 
* WinUI2_GettingStarted

Existing folder names, listed at https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/get-started -
* WinUI3_GettingStarted
* WPF_GettingStarted
* WinForms_GettingStarted
* Win32_GettingStarted

### CODE REVIEW

File: GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  IgnorableNamespaces="uap mp">
+
+  <Identity
+    Name="7e053cd8-2f6f-46f0-b7bb-b27448358047"
+    Publisher="CN=collabera"
```
Comment (mikehoffms) [inline GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest:11]: @nishitha-burman @peiche-jessica - change to CN=microsoft?

todo: see what's in other project dirs .appmanifest files

File: GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="7e053cd8-2f6f-46f0-b7bb-b27448358047" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>MyUWPGetStartApp</DisplayName>
+    <PublisherDisplayName>collabera</PublisherDisplayName>
```
Comment (mikehoffms) [inline GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest:18]: @nishitha-burman @peiche-jessica - change to microsoft? --

```
<PublisherDisplayName>microsoft</PublisherDisplayName>
```

todo: see what's in other project dirs .appmanifest files

File: GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest (hunk 1 above)
Comment (mikehoffms) [reply GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest:11]: @jasonstephen15 - Delete this "Package.appxmanifest" file?  Most projects in this repo seem to omit it (checking).

Comment (mikehoffms) [reply GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest]: I changed to "Microsoft" in the two spots in this file, built & ran ok.  So I might Resolve thread.

File: GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest (hunk 2 above)
Comment (mikehoffms) [reply GettingStartedGuides/WinUI2_GettingStarted/MyUWPGetStartApp/Package.appxmanifest]: Changed to "Microsoft" 2x, built/ran ok.

**Review by nishitha-burman — APPROVED** [review]

**Review by peiche-jessica — APPROVED** [review]
Thanks @mikehoffms for the `MyUWPGetStartApp.sln` naming suggestions, are we incorporating the naming changes in this PR or in the future?

**Pull request merged**